src/services/onchain-data.service.ts

```diff
@@ -197,6 +198,26 @@ export class OnchainDataService implements OnModuleInit
         }
     }
 
+    private filterNonStandardPairsTokens(): void
+    {
+        for (const pairAddress in this.pairs)
+        {
+            // Filters out pairs where either one token is an LP token
```

Comment:
Why is this necessary? An LP token is just one of many types of un-interesting tokens out there...

Reply:
I thought that's what we wanted to do? Didn't we agree that we'd only use the whitelist on the `info` frontend but not for the API?

Reply:
But why are LP tokens not interesting? This seems very arbitrary...

Reply:
yeah it is kind of arbitrary. So we shouldn't filter out pairs at all? And we'll just filter out tokens that don't have a USD price from our `/tokens`endpoint?

Reply:
Correct, that would be my vote. Filtering tokens without a usdPrice is much less arbitrary & protects developers

Reply:
Okay fixed. No longer filtering pairs, only tokens.